### ISSUE TITLE
[Onboarding] Epic

### ISSUE THREAD

**Issue by flexsurfer** [status-im/status-react#8089]
**In progress**

- [ ] @siphiuel   [**BUG**][Onboarding] Design touch-ups #8551 

**Not started**

- [ ] [Onboarding] Integrate fingerprint functionality #8533 
- [ ]  [Onboarding] Welcome screens #8135
        Features:  New design and UX

**Postponed**

- [ ] [Onboarding] Store keys with the 6-digit pincode #8535 - **blocked by status-go, work in progress**
        Features:  New design, UX and new way to store keys with 6-digit pincode
        **Impact**:  status-go, **security** https://github.com/status-im/status-go/issues/1456, https://github.com/status-im/status-go/issues/1463

**Done**

- [x] @pacamara [bounty][Onboarding] Recovery #8136
        Features:  New design

- [x]  [Onboarding] Splash screen #8130
        Features:  New design

- [x]  [Onboarding] Get started screens #8131 PR https://github.com/status-im/status-react/pull/8234
        Features:  New design and UX

- [x]  [Onboarding] Generate keys #8132   PR https://github.com/status-im/status-react/pull/8234
        Features:  New design, UX and the new way how to generate and choose an account
        **Impact**:  status-go, **security** https://github.com/status-im/status-go/issues/1456

- [x]  [Onboarding] Select storage and save with password #8134
        Features:  New design, UX 

- [x] [Onboarding] Have the ability to use fingerprint log in to the app (including Touch ID) #6573 [BOUNTY] PR https://github.com/status-im/status-react/pull/8127

- [x] [Onboarding] Update sign-in flow #8032 
        Features:  New design

- [x] [Onboarding] Create new account flow #8540 

- [x] [**BUG**][Onboarding] Notifications settings isn't shown #8544
